haskell/haskell.bzl

```diff
@@ -226,6 +227,42 @@ $ bazel-bin/.../hello-lib-repl  # run the script
 ```
 """
 
+def _haskell_prebuilt_library(ctx):
+  return [HaskellPrebuiltPackageInfo(
+            package = ctx.attr.package)]
+
+"""Wrap a prebuilt dependency as a rule.
+
+Example:
+  ```bzl
+  haskell_prebuilt_library(
+      name = "base_pkg",
```

Comment:
There seems to be typo in this example, since you name the target `"base_pkg"` by the rule below depends on `":base"`.

Reply:
Done.